tactical-microgrid-standard/controller/rtps_disc.ini

```diff
@@ -0,0 +1,14 @@
+[common]
+DCPSGlobalTransportConfig=$file
+DCPSSecurity=0
+
+[domain/11]
+DiscoveryConfig=uni_rtps
+
+[rtps_discovery/uni_rtps]
+SedpMulticast=0
+ResendPeriod=1
+
+[transport/the_rtps_transport]
+transport_type=rtps_udp
+use_multicast=0
```

Comment:
FYI I changed `Handshaking` to default to RTPS/UDP. It still uses otherwise default settings for that though, so it's different than what this config file does.

https://github.com/OpenDDS/DeveloperResources/blob/505e6e090818cfa1885496f3846c84bb7bd0a55f/tactical-microgrid-standard/common/Handshaking.cpp#L26-L32